From 82facd02eaf872de4bdea1d7251697bfb1f8650a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Thu, 4 Jul 2024 07:38:27 +0200
Subject: [PATCH] Add AdwaitaLegacy to themeNames

Most of the non-symbolic icons were moved from Adwaita to AdwaitaLegacy, but some legacy applications still use them.
---
 src/asgen/iconhandler.d | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/asgen/iconhandler.d b/src/asgen/iconhandler.d
index 562869a..e5b1a8e 100644
--- a/src/asgen/iconhandler.d
+++ b/src/asgen/iconhandler.d
@@ -340,6 +340,7 @@ public:
         if (iconTheme !is null)
             themeNames ~= iconTheme;
         themeNames ~= "Adwaita"; // GNOME
+        themeNames ~= "AdwaitaLegacy"; // GNOME
         themeNames ~= "breeze"; // KDE
 
         Package getPackage (const string pkid)
